CONTRIBUTING.md

```diff
@@ -51,6 +51,44 @@ Work on your self-assigned issue and eventually create a Pull Request.
    pre-commit install
    ```
 
+1. On MacOS with Apple Silicon, if this installation approach runs into errors, you may need to install the following prerequisites:
+
+   ```bash
+   brew install cmake libomp
+   ```
+
+which require `homebrew` to be installed on your development machine.  Please install these libraries and then repeat the previous step:
+
+```bash
+pip install -e .
```

Comment:
this should probably be .[test]

Reply:
@w4nderlust Yes -- what was happening is that this command would not work in Z-shell; but now I figured it out -- the "[test]" has to be in quotes (after several examples provided by @tgaddair I finally realized that this is what was happening).  Hence, in the next commit, I will tidy it all up and use these commands:
```
pip install -e ."[full]"
pip install -e ."[test]"
```

Thanks!